### PR TITLE
Add "get all user operations" and "rollback all user operations"

### PR DESCRIPTION
**Pull request by kaikue**
Still work in progress, PRing this now for feedback

### CODE REVIEW

File: pychunkedgraph/backend/graphoperation.py [hunk 1]
```diff
@@ -752,7 +752,7 @@ def __init__(
         multicut_as_split: bool,
     ) -> None:
         super().__init__(cg, user_id=user_id)
-        log_record = cg.read_log_row(superseded_operation_id)
+        log_record = cg.read_log_row(superseded_operation_id)[0] #TODO I don't know why this made it work...?
```
Comment (nkemnitz) [inline pychunkedgraph/backend/graphoperation.py:755]: The `read_log_row()` function return type was changed in 3d5bfaeadf2cf185887aacaed03a4c8c282a9d3f to additionally include a timestamp. This likely also broke other calls - at least the `RedoOperation.__init__`.

Comment (nkemnitz) [reply pychunkedgraph/backend/graphoperation.py]: The tests are failing for the same reason - I only added a mock for `read_log_row` back then, so now that one needs to be changed as well to include the time stamp.

Comment (kaikue) [reply pychunkedgraph/backend/graphoperation.py]: I just pushed a change adding the `[0]` de-timestamping to the other uses of `read_log_row()` in `graphoperation.py`. Should I change the tests too?

Comment (nkemnitz) [reply pychunkedgraph/backend/graphoperation.py]: That would be great, yes! Improvements and additions to the tests are also always welcome, but at least the few that we have should remain working.

**Comment by kaikue** [comment]
I think I messed up the rebase somehow...

File: pychunkedgraph/app/segmentation/v1/routes.py [hunk 2]
```diff
@@ -115,6 +116,33 @@ def handle_redo(table_id):
...
+    rollback_result = common.handle_rollback(table_id)
+    resp = rollback_result
+    return jsonify_with_kwargs(resp, int64_as_str=int64_as_str)
+
+
+### USER OPERATIONS -------------------------------------------------------------
+
+
+@bp.route("/table/<table_id>/user_operations", methods=["GET"])
+@auth_requires_permission("view") #TODO admin_view
```
Comment (sdorkenw) [inline pychunkedgraph/app/segmentation/v1/routes.py:135]: why not admin_view?

File: pychunkedgraph/app/segmentation/common.py [hunk 3]
```diff
@@ -544,6 +545,87 @@ def handle_redo(table_id):
...
+        except (cg_exceptions.PreconditionError, cg_exceptions.PostconditionError) as e:
+            raise cg_exceptions.BadRequest(str(e))
+
+        if ret.new_lvl2_ids.size > 0:
+            auth_header = {"Authorization": f"Bearer {current_app.config['AUTH_TOKEN']}"}
+            resp = requests.post(f"{current_app.config['MESHING_ENDPOINT']}/api/v1/table/{table_id}/remeshing",
+                                data=json.dumps({"new_lvl2_ids": ret.new_lvl2_ids},
+                                                cls=current_app.json_encoder), 
+                                headers=auth_header)
+            resp.raise_for_status()
```
Comment (sdorkenw) [inline pychunkedgraph/app/segmentation/common.py:583]: please use trigger_remesh here

**Pull request merged**